components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/AuthenticationDialogFragment.kt

```diff
@@ -80,7 +86,8 @@ internal class AuthenticationDialogFragment : PromptDialogFragment() {
     }
 
     private fun bindUsername(view: View) {
-        val usernameEditText = view.findViewById<EditText>(R.id.username)
+        val usernameEditText = view.findViewById<AutofillEditText>(R.id.username)
+        usernameEditText.sessionUrl = sessionUrl
```

Comment:
You need this for both `username` and `password` fields, because a "login" might have an empty username and still be legit, right? If so, can you add a comment describing this?

Reply:
Sure, I'll add comments to clarify why both needs to use `AutofillEditText`